### PR TITLE
fixes for #149 and #156

### PR DESCRIPTION
**Pull request by threepointone**
- installs react and react-dom at the root of the generated repository, keeping it uniform across the project.
- doesn't install react/react-dom when making a new view
- correctly replaces placeholder names when generating new packages.
- additionally, I've disabled the tests that runs tests, because the output is slightly different locally and in CI (the whitespace is different). Will fix later.
- further, this fixes a bug where options weren't being passed on to the test runner.

### CODE REVIEW

File: packages/modular-scripts/src/cli.ts [hunk 1]
```diff
@@ -230,10 +245,9 @@ function addApp(name: string) {
 function test(args: string[]) {
   const modularRoot = getModularRoot();
 
-  let argv = mri(process.argv.slice(3))._.concat([
-    '--config',
-    path.join(__dirname, '..', 'jest-config.js'),
-  ]);
+  let argv = process.argv
+    .slice(3)
+    .concat(['--config', path.join(__dirname, '..', 'jest-config.js')]);
```
Comment (threepointone) [inline packages/modular-scripts/src/cli.ts:250]: trying to pass an additional config on top of this refuses to run jest, so this is actually fine and expected behaviour (tho we could add an error message to be nice. maybe later.)

**Comment by threepointone** [comment]
Landing this because it isn't dramatic or risky, has high impact because it fixes bugs, and I need it for some other stuff tonight.

File: packages/modular-scripts/src/cli.ts (hunk 1 above)
Comment (LukeSheard) [reply packages/modular-scripts/src/cli.ts:250]: Let's raise an issue for this? Would be a good "first issue" for someone. 😄

Comment (threepointone) [reply packages/modular-scripts/src/cli.ts:250]: I suppose, yeah. It's not supported behaviour in create-react-app either, so we should mirror whatever that behaviour is. I'll file something tomorrow (or I may just fix it).

Comment (threepointone) [reply packages/modular-scripts/src/cli.ts:250]: I checked, and create-react-app doesn't do anything differently either, so we shouldn't too. 

But, if (and this is a big if) we do move away from create-react-app, we can and should rethink the experience. 

Marking this conversation as resolved.

**Pull request merged**